riff-raff/app/assets/javascripts/form-autocomplete.coffee

```diff
@@ -4,6 +4,23 @@ menuOpen = false
 updateBuildInfo = (buildNumber) ->
   $('#build-info').load(jsRoutes.controllers.DeployController.buildInfo(selectedProject, buildNumber).url)
 
+updateStageInfo = () ->
+  elemProjectInput = $('#projectInput')
+  isExactMatch = elemProjectInput.hasClass("project-exact-match")
+  selectedProject = elemProjectInput.val()
+  selectedBuild = $('#buildInput').val()
+
+  url = jsRoutes.controllers.DeployController.allowedStages(selectedProject, selectedBuild).url
+
+  $.ajax({
+    url: url,
+    dataType: 'html',
+    success: (stageOptions) ->
+      stageInput = document.getElementById('stage')
```

Comment:
minor: It looks like we use jquery across the repo? Should we use that here too for consistency? (albeit not very "modern").

Reply:
👍 this makes me sad 😢 but perhaps I should be more grateful for jquery in my life.